### PR TITLE
Nextlevel

### PR DESCRIPTION
**Pull request by XerxesNoble**

### CODE REVIEW

File: src/audio/index.js [hunk 1]
```diff
@@ -20,8 +20,14 @@ export default {
     }
   },
   jump() {
+    this.play([25])
+  },
+  win() {
     this.play([25,20,15])
   },
+  powerup() {
+    this.play([13,12,11,10], 0.08)
```
Comment (Fionoble) [inline src/audio/index.js]: Nice lol

File: src/engine.js [hunk 2]
```diff
@@ -5,12 +5,34 @@ import { EVENTS } from './constants'
...
+    this.hud = hud
+    this.context = context
+    this.canvas = context.canvas
+    this.currentLevel = 0
+
+    // When player reaches goal, Go to next map
+    this.canvas.addEventListener(EVENTS.LEVEL_COMPLETE, () => {
+      this.currentLevel++
+      if (this.currentLevel === map.levels.length) {
+        // YOU WIN!
```
Comment (Fionoble) [inline src/engine.js]: I'll get this screen going soon too. Gonna get assets this weekend

File: src/index.js [hunk 3]
```diff
@@ -18,18 +20,25 @@ const game = {
   },
   gameFailSequence() {
     this.canvas.removeEventListener(EVENTS.LEVEL_FAIL, this._gameFailSequence);
-    console.log('Yo, don\'t suck');
     audio.die()
     this.engine.stop();
     // TODO - Display Retry Screen
     setTimeout(() => window.location.reload(), 3000);
+  },
+  frame() {
```
Comment (Fionoble) [inline src/index.js]: What's the purpose of this function?

File: src/map/levels/level-2.js [hunk 4]
```diff
@@ -1,12 +1,13 @@
...
-  '    #         %                                         ',
-  '** ***  ** ******* ******* ***  * ***     ** ***   *****',
-  ' P    **          **    %              #         %      ',
-  '***** *  *** ******* ******* **  *** *******  ******* **',
+  '                                                        ',
+  '                                                        ',
+  '                                                        ',
+  '                                                        ',
+  '********************************************************',
+  ' P                                 $                     ',
```
Comment (Fionoble) [inline src/map/levels/level-2.js]: Difficulty: `Insane`

File: src/index.js (hunk 3 above)
Comment (XerxesNoble) [reply src/index.js]: debugging... literally needed to walk frames to measure how to calc the gravity :)

File: src/map/levels/level-2.js (hunk 4 above)
Comment (XerxesNoble) [reply src/map/levels/level-2.js]: Difficulty: DEBUG

**Pull request merged**